pandas-stubs/core/apply.pyi

```diff
@@ -0,0 +1,242 @@
+import abc
+from abc import abstractmethod
+from collections.abc import (
+    Callable,
+    Hashable,
+    Iterable,
+    Iterator,
+    Sequence,
+)
+from typing import (
+    Any,
+    Generic,
+    Literal,
+    TypeVar,
+)
+
+import numpy as np
+from pandas import (
+    DataFrame,
+    Index,
+    Series,
+)
+from pandas.core.generic import NDFrame
+from pandas.core.groupby import GroupBy
+from pandas.core.resample import Resampler
+from pandas.core.window.rolling import BaseWindow
+
+from pandas._libs.lib import NoDefault
+from pandas._typing import (
+    AggFuncType,
+    AggFuncTypeDict,
+    Axis,
+    AxisInt,
+    Incomplete,
+    NDFrameT,
+    npt,
+)
+from pandas.util._decorators import cache_readonly
+
+_AggObjT = TypeVar("_AggObjT", bound=NDFrame | GroupBy | BaseWindow | Resampler)
+_AggGroupByObjT = TypeVar("_AggGroupByObjT", bound=GroupBy | BaseWindow | Resampler)
+_AggResamplerWindowObjT = TypeVar(
+    "_AggResamplerWindowObjT", bound=BaseWindow | Resampler
+)
+
+ResType = dict[int, Any]  # noqa: PYI026
```

Comment:
Is that part of the public API? If not, it can/should be removed.

Reply:
Same also for all methods/attributes for all the classes.

If you cannot find a method/class here https://pandas.pydata.org/docs/reference/index.html, it is considered private. We still keep some private parent classes that are used by multiple public child classes.

It is already confusing what is private in pandas (most people expect that only _* are private), we do not want to add more confusion by highlighting private API in pandas-stubs. (That creates some inherent conflict with stubtest)

Reply:
Yeah I tried to fix everything stubtest complained about. We can add these symbols to the allowlist. I'll wait for the review from Dr-Irv and do a pass to handle those after if needed.